transcoders.go

```diff
@@ -373,3 +374,14 @@ func dnsStB(s string) ([]byte, error) {
 func dnsBtS(b []byte) (string, error) {
 	return string(b), nil
 }
+
+var TranscoderCertHash = NewTranscoderFromFunctions(certHashStB, certHashBtS, nil)
+
+func certHashStB(s string) ([]byte, error) {
+	_, data, err := multibase.Decode(s)
+	return data, err
+}
+
+func certHashBtS(b []byte) (string, error) {
+	return multibase.Encode(multibase.Base58BTC, b)
```

Comment:
This choice is totally arbitrary. We could use any other multibase that's widely supported.

Is there any best practice for choosing one over the other?

Reply:
cc @lidel, but the general guidelines I know of are:
- Do you want compact -> choose base64URL (`u`), although base58btc is probably fine too
- Do you want lowercase -> choose base32 (`b`)

Reply:
Switched to `Base64url`.